.pipelines/cni/load-test-templates/validate-state-template.yaml

```diff
@@ -12,9 +13,18 @@ steps:
       scriptType: "bash"
       addSpnToEnvironment: true
       inlineScript: |
-        make -C ./hack/aks set-kubeconf AZCLI=az CLUSTER=${{ parameters.clusterName }}-$(make revision)
+        make -C ./hack/aks set-kubeconf AZCLI=az CLUSTER=${{ parameters.clusterName }}
         kubectl get pods -A
         make test-validate-state OS=${{ parameters.os }} RESTART_CASE=${{ parameters.restartCase }} CNI_TYPE=${{ parameters.cni }}
+
+        # If this step fails, test manually

```

Comment:
nit: should we add this to the pipeline logs ? i.e `echo` it out. So that when it fails, it shows in the pipeline log and easier to convey ?